plugins/inputs/jolokia/jolokia.go

```diff
@@ -11,9 +11,14 @@ import (
 	"time"
 
 	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
+// Default http timeouts
+const DefaultResponseHeaderTimeout = 3
```

Comment:
just make these time.Duration objects directly, rather than converting to seconds below